### PR TITLE
List alias command

### PR DESCRIPTION
**Pull request by joycelynng**
Have not written test for list alias command.

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/ListAliasCommand.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+//@@author A0143756Y
+/**
+ * Lists all aliases in the alias manager to the user.
+ */
+public class ListAliasCommand extends Command {
+
+    public static final String COMMAND_WORD = "list-alias";
+    public static final String MESSAGE_SUCCESS = "Listed all aliases in alias manager.";
+
```
Comment (kabir-plod) [inline src/main/java/seedu/address/logic/commands/ListAliasCommand.java]: User doesn't need to know about alias manager

File: src/main/java/seedu/address/logic/commands/ListAliasCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    public ListAliasCommand() {}
+
+    @Override
+    public CommandResult execute() {
+        assert model != null;
+        
+        model.saveState();
+
```
Comment (kabir-plod) [inline src/main/java/seedu/address/logic/commands/ListAliasCommand.java]: State does not need to be saved as nothing is begin changed

Comment (joycelynng) [inline src/main/java/seedu/address/logic/commands/ListAliasCommand.java]: Resolved.

**Comment by KnytJC** [comment]
reviewed

**Pull request merged**